tests/testthat/test-markdown-code.R

```diff
@@ -1,33 +1,50 @@
-
-test_that("can eval", {
-  out1 <- roc_proc_text(rd_roclet(), "
-    #' @title Title `r 1 + 1`
-    #' @description Description `r 2 + 2`
-    #' @md
-    foo <- function() {}")[[1]]
-
+test_that("can eval inline code", {
+  example <- test_path("roxygen-block-md-code-simple-inline.R")
+  out1 <- roc_proc_text(rd_roclet(), brio::read_file(example))[[1]]
   expect_equal(out1$get_value("title"), "Title 2")
   expect_equal(out1$get_value("description"), "Description 4")
 })
 
-test_that("uses the same env for a block, but not across blocks", {
+test_that("can eval fenced code", {
+  example <- test_path("roxygen-block-md-code-simple-fenced.R")
```

Comment:
Understanding these tests is challenging because I have to move back and forth in PR and it's very hard to keep my place. Additionally, it also introduces a new style of testing which increases the net complexity of the roxygen2 tests, and adds overhead if I have to come back to this code years in the future. For this reason, I'd much prefer it if you'd keep the tests self-contained.